app/pages/sections/catalogue-solutions/price/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
+  "backLinkText": "Go back",
+  "title": "List price for",
+  "description": "Select the list price for this Catalogue Solution. If you’ve agreed a different price with the supplier, the values can be altered later.",
+  "listPriceHeading": "Select list price",
```

Comment:
remove this.

Reply:
why?

Reply:
I think it is right as there is a heading of "Select list price"